helpers/controller_helper.go

```diff
@@ -0,0 +1,34 @@
+package helpers
+
+import (
+	"regexp"
+
+	"github.com/beego/beego/v2/server/web"
+	"github.com/iancoleman/strcase"
+)
+
+type NestPreparer interface {
+	NestPrepare()
+}
+
+// SetControllerAttributes sets some attributes for controller
+func SetControllerAttributes(controller *web.Controller) {
+	controllerName, actionName := controller.GetControllerAndAction()
+
+	re := regexp.MustCompile(`Controller$`)
+	controllerName = re.ReplaceAllString(controllerName, "")
+
+	controller.Data["ControllerName"] = strcase.ToKebab(controllerName)
+	controller.Data["ActionName"] = strcase.ToKebab(actionName)
+}
+
+// SetFlashMessageLayout sets flash menssage layout for controller
+func SetFlashMessageLayout(controller *web.Controller) {
+	controller.LayoutSections = make(map[string]string)
+	controller.LayoutSections["FlashMessage"] = "shared/alert.html"
```

Comment:
@Lahphim Not sure about this but does the result different from adding the flash template on the layout directly?
```
{{ template "shared/alert.html" . }}
```

Reply:
@carryall Same approach ✌️
I decided to use this way because I think this will use the same format like this `{{ .LayoutContent }}`